packages/now-node/src/helpers.ts

```diff
@@ -49,16 +49,8 @@ function getBodyParser(req: NowRequest, body: Buffer) {
 function getQueryParser({ url = '/' }: NowRequest) {
   return function parseQuery(): NowRequestQuery {
     // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const { URL } = require('url');
-    // we provide a placeholder base url because we only want searchParams
-    const params = new URL(url, 'https://n').searchParams;
-
-    const query: { [key: string]: string | string[] } = {};
-    for (const [key, value] of params) {
-      query[key] = value;
-    }
-
-    return query;
+    const { parse: parseURL } = require('url');
+    return parseURL(url, true).query;
```

Comment:
We need a test 😃

Reply:
I decided to avoid using this method since it's deprecated (see https://nodejs.org/dist/latest-v12.x/docs/api/url.html#url_legacy_url_api). It doesn't look like it will be removed soon, so it should be fine (but we need a test 😄).

Reply:
The test can be added here: https://github.com/zeit/now/blob/631c7befad71a3ca71dd98f8635a8a46d4ec4936/packages/now-node/test/helpers.test.js#L134

Reply:
I don't think it'll ever be removed tbh. Node has a hard time actually removing old APIs, even if deprecated. To wit: `require('sys')` still works (`sys` was renamed to `util` _years_ ago) even on the latest version of Node.

_If_ there is a deprecation warning added to the program output then I'd agree about not using it, but since it's a "silent" deprecation then I think it's fine to still use this API.

Reply:
Added test case in 5c58abd.